api/model/storage/postgres/timeseries.go

```diff
@@ -781,6 +803,25 @@ func (f *TimeSeriesField) parseHistogram(rows pgx.Rows, mode api.SummaryMode) (*
 	}, nil
 }
 
+// For timeseries that exists as a single series in a file (ie. each row is just (timestamp, value))
+// we don't need to do any aggregation queries.  A single category is returned in the histogram,
+// and the exemplar name is set to the timeseries key.
+func (f *TimeSeriesField) generateUngroupedData() *api.Histogram {
+	return &api.Histogram{
+		Buckets: []*api.Bucket{
+			{
+				Key:   f.Key,
+				Count: 1,
+			},
+		},
+		Extrema: &api.Extrema{
+			Min: float64(1),
+			Max: float64(1),
+		},
```

Comment:
In this case, are the extrema supposed to be for the count, or for the underlying values?

Reply:
I think I actually need to go in and make sure that's being calculated properly - that was a placeholder that I meant to upated.

Reply:
Looks like this is correct - because it is limited to the single timeseries case (ie. sunspots) there is only every going to be one series, with a min and max of one for its bucket count.